src/main/java/frc/robot/Commands/ElevatorDirectProfileCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.Lib.MotionProfiles.DirectFeedProfile;
+import frc.Lib.MotionProfiles.DirectFeedProfile.DirectConfig;
+import frc.robot.Constants;
+import frc.robot.Subsystems.ElevatorSubsystem;
+
+//bind this command to a trigger in RobotContainer for testing
+public class ElevatorDirectProfileCommand extends CommandBase {
```

Comment:
This appears fine after just a quick spot check, but aren't we missing a pre-requisite here for the velocity control?